orchestrator/taste-orchestrator.py

```diff
@@ -889,12 +940,12 @@ def BuildCyclicSubsystems(cyclicSubsystems, cflagsSoFar):
         os.chdir("..")
 
 
-def RenameCommonlyNamedSymbols(scadeSubsystems, simulinkSubsystems, micropythonSubsystems, cSubsystems, cppSubsystems, adaSubsystems, rtdsSubsystems, ogSubsystems, guiSubsystems, cyclicSubsystems, vhdlSubsystems):
+def RenameCommonlyNamedSymbols(scadeSubsystems, simulinkSubsystems, micropythonSubsystems, cSubsystems, cppSubsystems, adaSubsystems, rtdsSubsystems, ogSubsystems, guiSubsystems, cyclicSubsystems, vhdlSubsystems, rosBridgeSubsystems):
     '''Identifies and renames identical symbols in separate subsystems'''
     g_stageLog.info("Renaming commonly named symbols")
 
     def getTarget(baseDir):
-        if baseDir in ogSubsystems or baseDir in guiSubsystems:
+        if baseDir in ogSubsystems or baseDir in guiSubsystems or baseDir in rosBridgeSubsystems:
```

Comment:
Could do as below `if baseDir in ogSubsystems + guiSubsystems + rosBridgeSubsystems:`

Reply:
ogSubsystems is a dict for some reason. But fixed.